packages/transaction-controller/CHANGELOG.md

```diff
@@ -15,6 +15,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - Added flow only will be activated if chainId is defined in feature flags.
 - Configure pending transaction polling intervals using remote feature flags ([#5549](https://github.com/MetaMask/core/pull/5549))
 
+### Changed
```

Comment:
This is the wrong place now, probably a bad merge.

But I also had comments concerning the formatting below, so maybe quickest to remove this and I can add back when we release?